### ISSUE TITLE
BrowserWindow accessibleTitle is not visible to screen readers

### ISSUE THREAD

**Issue by jchen437** [electron/electron#25981]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 9.0.4 
* **Operating System:**
  * macOS 10.15.7  
* **Last Known Working Electron version:**
  * Unknown 

### Expected Behavior

When the `accessibleTitle` property is set on a BrowserWindow, it is visible to the Mac Accessibility Inspector and is read aloud by screen readers such as VoiceOver.

### Actual Behavior

After setting the `accessibleTitle` property, it is not read when focusing on the window and using VoiceOver. I also can't find it anywhere using the Accessibility Inspector.

### To Reproduce

Small example can be found at https://github.com/jchen437/electron-quick-start (just forked and added `accessibleTitle` to the BrowserWindow in main.js).

To run:
```sh
$ git clone https://github.com/jchen437/electron-quick-start.git
$ cd electron-quick-start
$ npm install
$ npm start
```
